### PR TITLE
Add readme 1

### PR DESCRIPTION
**Pull request by kleberbaum**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently README doesn't serve it's purpose. #32 

**What is the new behavior (if this is a feature change)?**
- Now README serves it's purpose. #32 

**Other information**:
- Ref: https://help.github.com/en/articles/about-readmes

### CODE REVIEW

**Review by kleberbaum — COMMENTED** [review]
All peta clean and tidy. ^^

File: README.md [hunk 1]
```diff
@@ -1,6 +1,175 @@
...
+Now we're ready to set up the project itself:
+
+    pip install -r requirements/base.txt
+
+To set up your database and load initial data, run the following commands:
+
+    ./manage.py migrate
+    ./manage.py runserver
+
+## [](#status)Status
```
Comment (Aichnerc) [inline README.md]: This heading with its certain content makes no sense in this repository.

**Review by Aichnerc — CHANGES_REQUESTED** [review]
Please remove the website status, then it should be ok to merge.

File: README.md (hunk 1 above)
Comment (kleberbaum) [reply README.md]: yikes xD

File: README.md [hunk 2]
```diff
@@ -1,6 +1,171 @@
...
+  ·
+  <a href="https://github.com/aichner/Wagtail-Template/issues/new?template=feature_request.md">Request feature</a>
+  ·
+  <a href="https://www.aichner-christian.com/damn/dev">Blog</a>
+</p>
+
+## Table of contents
+
+- [Quick start](#quick-start)
+- [Status](#status)
```
Comment (Aichnerc) [inline README.md]: Please remove the reference to the removed section.

**Review by Aichnerc — CHANGES_REQUESTED** [review]
After this change, it should be okay to merge.

**Review by Aichnerc — CHANGES_REQUESTED** [review]
After this change, it should be okay to merge.

File: README.md (hunk 2 above)
Comment (kleberbaum) [reply README.md]: yikes xD

Comment (Aichnerc) [reply README.md]: image

**Review by Aichnerc — APPROVED** [review]
Should be okay to merge.

**Pull request merged**